docs/docs/user-guide/cmdlets/Update-M365DSCAzureAdApplication.md

```diff
@@ -32,22 +53,22 @@ This function does not generate any output.
 
 | Parameter | Required | DataType | Default Value | Allowed Values | Description |
 | --- | --- | --- | --- | --- | --- |
-| ApplicationName | False | String | Microsoft365DSC |  |  |
-| Permissions | True True | Hashtable[] |  |  |  |
-| Type | False | String | Secret | Secret, Certificate |  |
-| MonthsValid | False | Int32 | 12 |  |  |
-| CreateNewSecret | False | SwitchParameter |  |  |  |
-| CertificatePath | False | String |  |  |  |
-| CreateSelfSignedCertificate | False | SwitchParameter |  |  |  |
-| AdminConsent | False | SwitchParameter |  |  |  |
-| Credential | False | PSCredential |  |  |  |
-| ApplicationId | False | String |  |  |  |
-| TenantId | False | String |  |  |  |
-| ApplicationSecret | False | PSCredential |  |  |  |
-| CertificateThumbprint | False | String |  |  |  |
-| ManagedIdentity | False | SwitchParameter |  |  |  |
+| ApplicationName | False | String | Microsoft365DSC |  | The name of the application to create or update. Default value is 'Microsoft365DSC'. |
```

Comment:
You already have the default value in the nearby column?

However, the table looks really cramped. If this is not auto-generated from PowerShell sources, consider, maybe removing the "Default Value" column to give more space to Description?
```suggestion
| ApplicationName | False | String | Microsoft365DSC |  | The name of the application to create or update.|
```

Reply:
It's auto-generated...